src/lib/journey/journey.store.ts

```diff
@@ -89,6 +89,7 @@ export const journeyStore: Writable<JourneyStoreValue> = writable({
   step: null,
   successful: false,
   response: null,
+  recaptchaAction: null,
```

Comment:
Sorry, I came back to this after thinking on it for a bit. I'm not sure I understand this change to the store. Can you share why you're assigning `null` to every instance of this assignment?

Also, I feel like I'm missing something here. I don't see an additional property on the [interface for the options object of `journeyEvents.start`](https://github.com/ForgeRock/forgerock-web-login-framework/blob/beta/src/lib/widget/interfaces.ts#L23). I also don't see where we're falling back to the journey name if no action is passed in by the dev. I thought we'd iterate of the stack within the `journey.store` and find the first journey name.

Reply:
Seeing that you now added the property to the API utility module, I think this is now covered.